UefiPayloadPkg/UniversalPayloadBuild.py

```diff
@@ -130,10 +130,10 @@ def BuildUniversalPayload(Args):
     BuildDir     = os.path.join(os.environ['WORKSPACE'], os.path.normpath("Build/UefiPayloadPkg{}").format (Args.Arch))
     if Args.Arch == 'X64':
         BuildArch      = "X64"
-        FitArch        = "x86_64"
+        FitArch        = "i386"

```

Comment:
As I said on the other change, this shouldn't be here. coreboot will call payloads in 32-bit mode no matter what, so coreboot will always pass `-a IA32` to the script